src/CP/Breadcrumbs.php

```diff
@@ -37,7 +37,8 @@ public function jsonSerialize()
 
     public function title($title = null)
     {
-        $crumbs = $this->crumbs->map->text;
+        $new_crumbs = $this->crumbs->map(fn ($v) => ['text' => __($v['text']), 'url' => $v['url']]);
```

Comment:
maybe simplify to `$crumbs = $this->crumbs->map(fn ($v) => __($v['text']))` ?

Reply:
Nice!